json_serializable/lib/src/type_helpers/json_converter_helper.dart

```diff
@@ -76,7 +76,12 @@ _JsonConvertData _typeConverter(DartType targetType, TypeHelperContext ctx) {
   var matchingAnnotations = converterMatches(ctx.fieldElement.metadata);
 
   if (matchingAnnotations.isEmpty) {
-    matchingAnnotations = converterMatches(ctx.classElement.metadata);
+    matchingAnnotations =
+        converterMatches(ctx.fieldElement.getter?.metadata ?? []);
+
+    if (matchingAnnotations.isEmpty) {
```

Comment:
I might have avoided the nesting, no need to fix if travis is happy but if we need to push another commit we may as well unnest these.

Reply:
I agree to this from a code style point of view, although I am pondering if it'd have any runtime effect.

Reply:
Looks like Travis is green.  Happy to push another commit if you'd like?

Reply:
probably a slight runtime effect, but indentation scares me 😝 

this is fine as is.